fuzz/mkfuzzoids.pl

```diff
@@ -1,15 +1,17 @@
 #! /usr/bin/env perl
-# Copyright 1995-2020 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 1995-2021 The OpenSSL Project Authors. All Rights Reserved.
 #
 # Licensed under the Apache License 2.0 (the "License").  You may not use
 # this file except in compliance with the License.  You can obtain a copy
 # in the file LICENSE in the source distribution or at
 # https://www.openssl.org/source/license.html
+use FindBin;
+use lib "$FindBin::Bin/../util/perl";
+use OpenSSL::copyright;
 
 my $obj_dat_h = $ARGV[0];
-
-# The year the output file is generated.
-my $YEAR = [localtime()]->[5] + 1900;
+# Not used in the output, unfortunately.
+my $YEAR = OpenSSL::copyright::latest(($0, $obj_dat_h));
```

Comment:
Am I missing something? I can't see how this is used at all....can we just delete this line?

Reply:
I was hoping to update the output file to have comments.  But for now, I can put comment-out that line.

Reply:
It turns out that oss-fuzz allows comments, so I updated the script to output the common copyright notice and year.